### PR TITLE
Topic 2

### PR DESCRIPTION
**Pull request by marioylink11**

### CODE REVIEW

**Comment by troianoandres** [comment]
Please fix this PR to not have the CSS stuff inside. THere are 22 files to review. Thanks

**Comment by marioylink11** [comment]
My first push/PR in Topic 2 was  "Erasing Everything
<https://github.com/marioylink11/startup/pull/4/commits/509a217f341a2247af8c81ab33bf1896bc261d12>"
where i deleted ALL the old files (because idk why there were old files in
there). I think there is a few files (the necessary ones) right now in the
latter PRs.

I listed all the files in Topic 2 and the result was this:





2018-05-21 9:41 GMT-03:00 Troiano Andres <notifications@github.com>:

**Comment by troianoandres** [comment]
You will delete the CSS stuff if you merge this. Fix it please

File: Topic2/1,2,3and4/1234.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+		this.duration = duration;
+		this.cast = [];
+	}
+	
+	play(){
+		console.log("the movie is playing ");
+	}
+	
+	addCast(cast){
+	    if (cast instanceof Actor) {
```
Comment (troianoandres) [inline Topic2/1,2,3and4/1234.js]: Will this work? Don't you need to add the for statement in an else?

**Comment by marioylink11** [comment]
I want to delete the CSS files off this branch. What is the problem?

**Pull request closed without merging**